component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/TOTPAuthenticatorConstants.java

```diff
@@ -91,7 +91,7 @@ private TOTPAuthenticatorConstants() {
 	public static final String PROPERTY_ACCOUNT_LOCK_TIME = "account.lock.handler.Time";
```

Comment:
open a public issue in product-is and say that it gets resolved with this PR. Mention this PR in the issue too